### ISSUE TITLE
Size of output image --w --h arguments

### ISSUE THREAD

**Issue by m17358m** [CompVis/stable-diffusion#89]
Hi! Everything is working perfectly, I love it, thanks a lot! :) when using img2img, I want an output bigger than the input image. Is that possible? I added
     parser.add_argument(
        "--H",
        type=int,
        default=None,
        help="image height, in pixel space",
    )
    
    parser.add_argument(
        "--W",
        type=int,
        default=None,
        help="image width, in pixel space",
    )

but with no effect on the output (still same as input, 512x512). Is there some way to achieve this? thanks in advance!

**Comment by junukwon7** [comment]
I'm not sure, but the stable diffusion model (especially v1) is trained at 512x512 so changing might not be a good idea. (Maybe not)

Temporary alternative: till someone appears and makes the code resizable, try using upscalers like [Nero Upscaler](https://ai.nero.com/image-upscaler) or [Gigapixel](https://www.topazlabs.com/gigapixel-ai)

**Comment by m17358m** [comment]
Yes, doing it with Gigapixel :) I'm using a Tesla T4 (16gb) and that's not enough for 1024x1024 (runs out of mem), so I'll try with 24gb video memory next, that should do the trick.

**Comment by rcarneva** [comment]
It's fairly easy to just add a PIL resize into the script. I replaced line 316 of `txt2img.py` with 
```
img = Image.fromarray(x_sample.astype(np.uint8)).resize((1024, 1024), Image.NEAREST)
```
to get a double-sized output, and you could easily do something similar for `img2img.p;y`

**Comment by m17358m** [comment]
Ok, thanks for the info, I'll try doing that.
